### DISCUSSION TITLE
让模型显示影子shadows

### DISCUSSION THREAD

**Discussion by HHANG52** [zouyaoji/vue-cesium#461] [Q&A] [answered]
我根据文档在vc-viewer标签设置shadows=true 在模型中设置shadows=“3”，都没让模型显示出阴影。查阅百度也设置了
```javascript
 viewer.scene.globe.enableLighting = true
 viewer.shadows = true
```
第一次偶然显示出来，后续阴影都没显示出来怎么设置都没显示。是我漏了啥吗。求指导

**Comment by zouyaoji** [comment]
啥模型？

**Comment by HHANG52** [comment]
在地面上放置的model---我加载的一些glb模型。
VcGraphicsModel，加载模型实体，相当于初始化一个&nbsp;Cesium.ModelGraphics实例。



HHANG





------------------&nbsp;原始邮件&nbsp;------------------





 
啥模型？

**Comment by zouyaoji** [comment]
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/5cbf29e3-5e58-49b9-b4e6-5775feb5ada3)

![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/e36d3d1b-8e11-4608-8283-a3a6d332d295)

参考一下2个参数即可。

**Comment by HHANG52** [comment]
你好，很抱歉再次打扰，这么设置确实可以显示影子，经过一下午时间的开发，突然又这个阴影效果就没有了。设置一直都没变。就是不显示影子了。



&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zouyaoji/vue-cesium"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月26日(星期四) 中午11:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zouyaoji/vue-cesium] 让模型显示影子shadows (Discussion #461)





 

 

 
参考一下2个参数即可。

**Reply by zouyaoji** [reply]
不知道了，估计你改到啥了

**Comment by HHANG52** [comment, marked as answer]
找到了原因，受计算机系统时间影响，在某些时候没有光照，就没有太阳光。修改计算机系统时间，影子又出来了。
&nbsp;


HHANG
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月7日(星期二) 下午2:06
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zouyaoji/vue-cesium] 让模型显示影子shadows (Discussion #461)





 
不知道了，估计你改到啥了
